src/app/model/instance.ts

```diff
@@ -0,0 +1,19 @@
+import { inheritSerialization, autoserialize } from 'cerialize';
+
+import { Model } from './model';
+
+import { Geolocation } from '@ionic-native/geolocation/ngx';
+
+@inheritSerialization(Model)
+export class Instance extends Model {
+  @autoserialize public counterId: string;
+  @autoserialize public count: number;
```

Comment:
`Instance` fingi chli sehr generisch, wottme das nid irgendwie `CounterEvent` oder so toufe? z attribut `count` würdi villecht ou irgendwie zu `offset` oder `delta` oder so umbenenne.

Reply:
Ja finge dr name o nid ideau, gaern offe fuer vorschlaeg. `CountEvent` finig aber no viu schlechter da das wuerd suggeirer das es irgend oeppis mit em ne event handler z due het.

Reply:
`CountInstance` ?

Reply:
rename `Instance` to `CountInstance` and rename the attribute `count` to `change`

Reply:
contrary to my last comment I decided and changed this to `CountEvent`. On deeper thought I kind of like the name.

Reply:
also I will keep `count` and introduce a new attribute `delta`
* delta: holds what currently is called `count`, namely a chnage of the counter
* count: the state of the counter after the change in `delta`

reason why I will do this that way is because, I just implemented a reset button. Which will add a Count event with `count = 0` and `delta = 0`.